File: sdks/python/fiduscan/blockchain.py

class BlockchainClient:
    def __init__(self, client):
        self.client = client

    def create_anchor(self, evidence_id):
        return self.client.request("POST", "/blockchain/anchor", json={"evidence_id": evidence_id})

    def verify_anchor(self, anchor_id):
        return self.client.request("GET", f"/blockchain/anchor/{anchor_id}/verify")

File: sdks/node/src/modules/blockchain.ts

import { Client } from "../client";

export class BlockchainClient {
  constructor(private client: Client) {}

  async createAnchor(evidenceId: string) {
    return this.client.request("POST", "/blockchain/anchor", { evidence_id: evidenceId });
  }

  async verifyAnchor(anchorId: string) {
    return this.client.request("GET", `/blockchain/anchor/${anchorId}/verify`);
  }
}
